index.html

```diff
@@ -60,5 +60,156 @@ <h1 id="introduction" class="poppins-font">
       </ul>
     </div>
   </section>
+
+  <!-- work cards -->
+  <div id="projets">
+    <!-- card 1 -->
+    <section class="container-bg-color project">
+     <img src="./images/img1.svg" alt="protfolio snapshot">
+     <h3 class="proj-title">Tonic</h3>
+     <ul class="proj-about poppins-font no-bullets">
+      <li class="proj-name ">CANOPY</li>
+      <li class="proj-type">
+        Back End Dev</li>
+      <li class="proj-year">
+        2022</li>
+     </ul>
+     <p class="proj-primarytxt poppins-font">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+     <ul class="no-bullets proj-list poppins-font">
+      <li>html</li>
+      <li>css</li>
+      <li>javascript</li>
+     </ul>
+     <button type="button" class="poppins-font button">See Project</button>
```

Comment:
- [OPTIONAL] I suggest following the button styling: Active, Hover and Pressed

> Note: Please don't forget to do this on all buttons 👌

![2](https://user-images.githubusercontent.com/45231276/176837061-facdfea9-a844-49bc-bed7-93be8ab7b49c.PNG)